docs/gettingstarted/sc4snmp-installation.md

```diff
@@ -42,6 +42,22 @@ or download it directly from Helm using the command `microk8s helm3 show values
 
 It is recommended to start by completing the base template and gradually add additional configurations as needed.
 
+
+The `values.yaml` file is validated using `JSON schema` built into `helm chart` and additionally validated inside the code.
```

Comment:
and additionally validated inside the code. -> and inside the code.

Reply:
Done